libs/partners/upstage/langchain_upstage/parsers.py

```diff
@@ -0,0 +1,342 @@
+            f" `{env_key}` which contains it, or pass"
+            f"  `{key}` as a named parameter."
+        )
+
+
+class LayoutAnalysisParser(BaseBlobParser):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        output_type: OutputType = "text",
+        split: SplitType = "none",
+    ):
+        """
+        Initializes an instance of the Upstage class.
+
+        Args:
+            output_type (OutputType, optional): The desired output type.
+                                                Defaults to "text".
+            split (SplitType, optional): The type of splitting to be applied.
+                                         Defaults to "none".
+            api_key (str, optional): The API key for accessing the Upstage API.
+                                     Defaults to None, in which case it will be
+                                     fetched from the environment variable
+                                     `UPSTAGE_API_KEY`.
+        """
+        self.api_key = get_from_param_or_env(
+            "UPSTAGE_API_KEY", api_key, "UPSTAGE_API_KEY"
+        )
+
+        self.output_type = output_type
+        self.split = split
+
+        validate_api_key(self.api_key)
+
+    def _get_response(self, files: Dict) -> Dict:
+        """
+        Sends a POST request to the API endpoint with the provided files and
+        returns the response.
+
+        Args:
+            files (dict): A dictionary containing the files to be sent in the request.
+
+        Returns:
+            dict: The JSON response from the API.
+
+        Raises:
+            ValueError: If there is an error in the API call.
+        """
+        try:
+            headers = {"Authorization": f"Bearer {self.api_key}"}
+            options = {"ocr": False}
+            response = requests.post(
+                LAYOUT_ANALYZER_URL, headers=headers, files=files, json=options
+            )
+            response.raise_for_status()  # HTTP 요청이 성공적으로 완료되었는지 확인
+
+            try:
+                result = response.json()  # JSON 디코딩
+            except json.JSONDecodeError as e:
+                raise ValueError(f"Failed to decode JSON response: {e}")
+        finally:
+            if "document" in files and not files["document"].closed:
+                files["document"].close()
+
+        return result
+
+    def _split_and_request(
+        self,
+        full_docs: fitzDocument,
+        start_page: int,
+        split_pages: int = LIMIT_OF_PAGE_REQUEST,
+    ) -> Dict:
+        """
+        Splits the full pdf document into partial pages and sends a request to the
+        server.
+
+        Args:
+            full_docs (str): The full document to be split and requested.
+            start_page (int): The starting page number for splitting the document.
+            split_pages (int, optional): The number of pages to split the document into.
+                                         Defaults to LIMIT_OF_PAGE_REQUEST.
```

Comment:
```suggestion
                                         Defaults to DEFAULT_PAGE_BATCH_SIZE.
```

위 naming 제안 드린것과 맞추기위해..